elements/pl-symbolic-input/pl-symbolic-input.mustache

```diff
@@ -67,21 +85,29 @@ ${{a_tru}}$
 {{/answer}}
 
 {{#format}}
-Your answer must be a symbolic expression. All numbers must be rational - so, <code>1/2</code> instead of <code>0.5</code>.
-{{^allow_complex}}Complex numbers are not allowed.{{/allow_complex}}
-<hr>
+<p>Your answer must be a symbolic expression. All numbers must be rational - so, <code>1/2</code> instead of <code>0.5</code>.
+{{^allow_complex}}Complex numbers are not allowed.{{/allow_complex}}</p>
+    
 <h5><small class='text-uppercase'>Allowable Operators</small></h5>
```

Comment:
The uppercase headers feel a bit shouty to me, and we aren't using them anywhere else. Perhaps change to just regular `<h5>` or even `<h4>`?

Reply:
<img width="485" alt="image" src="https://user-images.githubusercontent.com/1790491/75059303-4ce91400-54a2-11ea-8d80-c1403bfa3532.png">

Here they are with `<h6>`, `5` and `4` felt too "big" to me.  Thoughts?

Reply:
I like this. The listing of operators/constants/values after the headings feels a bit weird, in that the headings are really the name of the list that's on the next line, if that makes sense?

I wonder if this would be clearer if we grouped variables/constants/operators together? I also think that maybe the order should be variables/constants/operators in terms of which ones are more likely to change from question to question? And somehow make the "Allowable X" more of labels than headings?

Just as an untested example (I have no idea whether this looks any good):
```
<dl>
    <dt>Allowable variables:</dt>
    <dd>a, b, y</dd>

    <dt>Allowable constants:</dt>
    <dd>e, pi</dd>

    <dt>Allowable operators:</dt>
    <dd>cos, sin, tan, ...</dd>
</dl>

All operations must be explicit:
...
```

Reply:
![image](https://user-images.githubusercontent.com/1790491/75074296-f808c600-54c0-11ea-9210-e25ff5b1c66f.png)
something like this?

Reply:
Definitely clearer, although I much confess that `class="user-output"` is a bit overwhelming 😄 --- I think it's a net win for clarity, however.

What about putting the variables/constants/operators on the same lines as "Allowable variables", etc?

Reply:
![image](https://user-images.githubusercontent.com/1790491/75075370-44ed9c00-54c3-11ea-9671-3757066715dd.png)

Saves quite a bit of vertical space which is nice.  I could go either way here, really.

Reply:
(wait hold up I just realised what you meant)

Reply:
![image](https://user-images.githubusercontent.com/1790491/75075618-dd841c00-54c3-11ea-9797-0b04528b7c7d.png)
This looks a bit unbalanced, imo.  I would prefer one of the two previous designs.

Reply:
Agreed. I personally like the immediately-previous design the best (actually the second-previous, skipping the obviously broken one 😄)